samples/gltf_viewer.cpp

```diff
@@ -893,6 +893,8 @@ int main(int argc, char** argv) {
 
     auto cleanup = [&app](Engine* engine, View*, Scene*) {
         app.automationEngine->terminate();
+        app.resourceLoader->asyncCancelLoad();
```

Comment:
Should the resource loader be responsible for the `flushAndWait`?

Reply:
That's a good idea, will make this change.